packages/forms/src/UIForm/fields/Datalist/Datalist.component.test.js

```diff
@@ -56,7 +62,7 @@ describe('Datalist component', () => {
 	it('should render', () => {
 		// when
 		const wrapper = shallow(
-			<Datalist.WrappedComponent
+			<Datalist
```

Comment:
With the talend/scripts react-i18next mock, we don't have the WrappedComponent anymore. The HOC returns the component directly with the `t()` function as default props to simulate the injection.

We have to remove all the `.WrappedComponent` from tests